ILRepack/Steps/AttributesRepackStep.cs

```diff
@@ -88,7 +90,7 @@ void CleanupAttributes()
             {
                 String name = (string)ca.ConstructorArguments[0].Value;
                 int idx;
-                if ((idx = name.IndexOf(", PublicKey=", StringComparison.Ordinal)) != -1)
+                if ((idx = name.IndexOfRegex(new Regex(", ?PublicKey="))) != -1)
```

Comment:
while it perfectly does the trick here, the allowed format is more permissive, e.g. you can have quotes around the value, or spaces around the '=', and the 'publickey' case doesn't matter.
I thought using `new AssemblyName(str).GetPublicKeyToken() != null` would work just fine w/o us having to care about the format details, but it doesn't like `PublicKey=null` which seems to be valid in `InternalsVisibleTo` references... :sob: